src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,98 @@
+package duke;
+
+import duke.commands.*;
```

Comment:
As per the coding standard, all imported classes should be listed explicitly. I think this issue appears in a couple of other files too. You might want to configure your IDE if it is automatically converting multiple imports into wildcard imports.